models/job_offer.js

```diff
@@ -18,8 +18,24 @@ jobSchema = mongoose.Schema({
         type: String,
         required: true
     }],
-    soft_skills: [String],
-    hard_skills: [String],
+    soft_skills: [{
+      name: String,
+      importance: {
+        type: Number,
+        min : 1,
+        max : 4,
+        required : true
+      }
+    }],
+    hard_skills: [{
+      name: String,
+      importance: {
+        type: Number,
+        min : 1,
+        max : 4,
+        required : true
+      }
+    }],
```

Comment:
Shouldn't there also be something similar to work_culture_preferences in job_offer? I thought we wanted both to have the same fields more or less